recovery/translation_nl.ts

```diff
@@ -510,128 +510,128 @@ De instelling opslaan?</translation>
         <location filename="multiimagewritethread.cpp" line="413"/>
         <source>%1: Writing OS image</source>
         <oldsource>Writing image</oldsource>
-        <translation type="unfinished">Wegschrijven image</translation>
+        <translation>Wegschrijven image</translation>
     </message>
     <message>
         <location filename="multiimagewritethread.cpp" line="427"/>
         <source>%1: Mounting file system</source>
         <oldsource>Mounting file system</oldsource>
-        <translation type="unfinished">Mounten bestandssysteem</translation>
+        <translation>Koppelen van bestandssysteem</translation>
     </message>
     <message>
         <location filename="multiimagewritethread.cpp" line="435"/>
         <location filename="multiimagewritethread.cpp" line="459"/>
         <source>%1: Error mounting file system</source>
         <oldsource>Error mounting file system</oldsource>
-        <translation type="unfinished">Fout bij mounten bestandssysteem</translation>
+        <translation>Fout bij mounten bestandssysteem</translation>
     </message>
     <message>
         <location filename="multiimagewritethread.cpp" line="420"/>
         <source>%1: Creating filesystem (%2)</source>
-        <translation type="unfinished">Aanmaken bestandssysteem (%1) {1:?} {2)?}</translation>
+        <translation>Aanmaken bestandssysteem (%1) {1:?} {2)?}</translation>
     </message>
     <message>
         <location filename="multiimagewritethread.cpp" line="65"/>
         <source>RISCOS cannot be installed. RISCOS offset value mismatch.</source>
-        <translation type="unfinished"></translation>
+        <translation>RISCOS Kan niet installeren, RISCOS correctiewaarde klopt niet</translation>
```

Comment:
Wellicht beter om iets in de trend van "RISCOS kan niet geinstalleerd worden" te gebruiken. Het OS installeert zichzelf immers niet.